£Lerror_reason¢DtypeEvalueYHTTPSConnectionPool(host='support.ics.uci.edunone', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x00000000043024A8>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed',))Ihttp_code¢DtypeEvalueÛQdownload_complete¢DtypeEvalueı